manifests/c/CrystalIDEASoftware/UninstallTool/3.5.10/CrystalIDEASoftware.UninstallTool.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CrystalIDEASoftware.UninstallTool
+PackageVersion: 3.5.10
+PackageLocale: en-US
+Publisher: CrystalIDEA Software
+PackageName: Uninstall Tool
+License: Copyright © CrystalIDEA Software
+ShortDescription: Uninstall Tool Setup

```

Let's add a better description

```suggestion
ShortDescription: Fast and Powerful Uninstaller for Complete Removal of Programs.
```